e2e/cli-e2e/tests/__snapshots__/collect.spec.ts.snap

```diff
@@ -1,5 +1,1007 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
+exports[`CLI collect > should create report.md 1`] = `
```

Comment:
There's a new snapshot, but the `collect.spec.ts` test file wasn't changed, so I supposed it's just a leftover?

You can easily remove the obsolete snapshot with `npx nx e2e cli-e2e -u`.